src/pages/team/index.tsx

```diff
@@ -117,11 +121,12 @@ const AtCapacityNotice = ({teamData}: {teamData: TeamData}) => {
 }
 
 type TeamPageProps = {
-  team: TeamData | undefined
+  team: TeamData | null
 }
 
 const Team = ({team: teamData}: TeamPageProps) => {
   const router = useRouter()
+  const [members, setMembers] = React.useState<any[]>(teamData?.members || [])
```

Comment:
Using component state to manage the list of team members so that it can be updated with a `setMembers` call if someone is removed.